src/api/spec/controllers/person_controller_spec.rb

```diff
@@ -174,4 +195,59 @@
       end
     end
   end
+
+  describe 'GET #get_watchlist' do
+    context 'user logged-in' do
+      let(:xml) do
+        <<-XML_DATA
+          <watchlist>
+            <project name="test-proj"/>
+            <package name="test-pkg" project="test-proj"/>
+          </watchlist>
+        XML_DATA
+      end
+
+      before do
+        project = create(:project, name: 'test-proj')
+        package = create(:package, project: project, name: 'test-pkg')
+        Flipper.enable_actor(:new_watchlist, user)
```

Comment:
```suggestion
      let(:project) { create(:project, name: 'test-proj') }
      let(:package) { create(:package, project: project, name: 'test-pkg') }

      before do
        Flipper.enable_actor(:new_watchlist, user)
```

Reply:
We don't need this project and package outside `before` block :slightly_smiling_face:

Reply:
alright :+1: